libs/atomic-elements/src/lib/components/NavTop/NavTop.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { useState, MouseEvent, useRef } from 'react';
+
+import AppBar from '@material-ui/core/AppBar';
+import Avatar from '@material-ui/core/Avatar';
+import Button from '@material-ui/core/Button';
+import ClickAwayListener from '@material-ui/core/ClickAwayListener';
+import Grow from '@material-ui/core/Grow';
+import MenuItem from '@material-ui/core/MenuItem';
+import MenuList from '@material-ui/core/MenuList';
+import Paper from '@material-ui/core/Paper';
+import Popper, { PopperProps } from '@material-ui/core/Popper';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+
+import AccountCircleOutlinedIcon from '@material-ui/icons/AccountCircleOutlined';
+import HelpOutlineOutlinedIcon from '@material-ui/icons/HelpOutlineOutlined';
+import MenuIcon from '@material-ui/icons/Menu';
+
+import { object } from '@storybook/addon-knobs';
+
+import './NavTop.scss';
+
+export interface NavTopProps {
+  title: string;
+  menuList: MenuListProps[];
+  accountMenuList: MenuListProps[];
+  logo: string;
+}
+interface MenuListProps {
+  title: string;
+}
+
+interface MenuListSectionProps extends Partial<PopperProps> {
+  menuList: MenuListProps[];
+  handleClose: (value: unknown) => void;
```

Comment:
```suggestion
  handleClose: (event: unknown) => void;
```
or 

```
  handleClose: () => void;
```

because I think what the event throws wouldn't be used